### PR TITLE
Doc: Add ARTEMIS Publications

### PR DESCRIPTION
**Pull request by ax3l**
Add an ARTEMIS paper :)

Congratulations @PeculiarOvertones, @jackieyao0114, @RevathiJambunathan, @ajnonaka et al.!! :tada:

### CODE REVIEW

File: Docs/source/highlights.rst [hunk 1]
```diff
@@ -65,9 +65,12 @@ Scientific works in particle and beam modeling.
...
 ****************
 
-.. note::
+`ARTEMIS (Adaptive mesh Refinement Time-domain ElectrodynaMIcs Solver) <https://ccse.lbl.gov/Research/Microelectronics/>`__ is based on WarpX and couples the Maxwell's equations implementation in WarpX with classical equations that describe quantum material behavior (such as, LLG equation for micromagnetics and London equation for superconducting materials) for quantifying the performance of `next-generation microelectronics <https://www.lbl.gov/research/microelectronics-and-beyond/>`__.
 
-  Link ARTEMIS publications.
+#. Sawant SS, Yao Z, Jambunathan R, Nonaka A.
+   **Characterization of Transmission Lines in Microelectronic Circuits Using the ARTEMIS Solver**.
+   IEEE Journal on Multiscale and Multiphysics Computational Techniques, vol. 8, pp. 31-39, 2023.
+   `DOI:10.1109/JMMCT.2022.3228281 <https://doi.org/10.1109/JMMCT.2022.3228281>`__
```
Comment (jackieyao0114) [inline Docs/source/highlights.rst:84]:
```suggestion
#. Kumar P, Nonaka A, Jambunathan R, Pahwa G and Salahuddin S, Yao Z. 
   **FerroX: A GPU-accelerated, 3D Phase-Field Simulation Framework for Modeling Ferroelectric Devices**. 
   arXiv preprint, 2022. 
   `arXiv:2210.15668 <https://doi.org/10.48550/arXiv.2210.15668>`__

#. Yao Z, Jambunathan R, Zeng Y, Nonaka A. 
   **A Massively Parallel Time-Domain Coupled Electrodynamics–Micromagnetics Solver**. 
   The International Journal of High Performance Computing Applications, 36(2):167-181, 2022.
   `DOI:10.1177/10943420211057906 <https://doi.org/10.1177/10943420211057906>`__

```

File: Docs/source/highlights.rst [hunk 2]
```diff
@@ -65,10 +65,21 @@ Scientific works in particle and beam modeling.
...
+#. Kumar P, Nonaka A, Jambunathan R, Pahwa G and Salahuddin S, Yao Z.
+   **FerroX: A GPU-accelerated, 3D Phase-Field Simulation Framework for Modeling Ferroelectric Devices**.
+   arXiv preprint, 2022.
+   `arXiv:2210.15668 <https://doi.org/10.48550/arXiv.2210.15668>`__
+
+#. Yao Z, Jambunathan R, Zeng Y, Nonaka A.
+   **A Massively Parallel Time-Domain Coupled Electrodynamics–Micromagnetics Solver**.
+   The International Journal of High Performance Computing Applications, 36(2):167-181, 2022.
+   `DOI:10.1177/10943420211057906 <https://doi.org/10.1177/10943420211057906>`__
```
Comment (ax3l) [inline Docs/source/highlights.rst:82]: Just source style:
```suggestion
   `DOI:10.1177/10943420211057906 <https://doi.org/10.1177/10943420211057906>`__

```

File: Docs/source/highlights.rst (hunk 1 above)
Comment (saurabh-s-sawant) [inline Docs/source/highlights.rst:84]: Hi @ax3l or @RevathiJambunathan could you please add a space between SS in the paper entitled,  "Characterization of Transmission Lines ...". Thanks!

File: Docs/source/highlights.rst [hunk 3]
```diff
@@ -65,9 +65,21 @@ Scientific works in particle and beam modeling.
 Microelectronics
 ****************
 
-.. note::
-
-  Link ARTEMIS publications.
+`ARTEMIS (Adaptive mesh Refinement Time-domain ElectrodynaMIcs Solver) <https://ccse.lbl.gov/Research/Microelectronics/>`__ is based on WarpX and couples the Maxwell's equations implementation in WarpX with classical equations that describe quantum material behavior (such as, LLG equation for micromagnetics and London equation for superconducting materials) for quantifying the performance of `next-generation microelectronics <https://www.lbl.gov/research/microelectronics-and-beyond/>`__.
+
+#. Sawant SS, Yao Z, Jambunathan R, Nonaka A.
```
Comment (ax3l) [inline Docs/source/highlights.rst]: Add space:
```suggestion
#. Sawant S S, Yao Z, Jambunathan R, Nonaka A.
```

File: Docs/source/highlights.rst (hunk 1 above)
Comment (ax3l) [reply Docs/source/highlights.rst:84]: Fixed, thanks! :)

**Review by RevathiJambunathan — APPROVED** [review]
Thanks for this PR, @ax3l !

**Pull request merged**